src/components/ErrorRun/index.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
```

Comment:
Yes it's a good way to test your new error boundaries.
But I suppose that the goal of this testing is a simulate an error INSIDE target component. Maybe it would be better to design it like custom hook and move to folder like "test_utils" for example? If it is custom hook, every one can import it to the every component for testing.

And I would very appreciate if you could write a comment about it and how to use it. 
Are you agree?

Reply:
I hadn't thought of that, it's a great idea. I'll implement it with a custom hook. Implementation. Write a custom hook which will wrap a component and if there is an error, it will display some message. The idea is this.